packages/main/src/Table.js

```diff
@@ -210,6 +210,10 @@ class Table extends UI5Element {
 		}
 	}
 
+	onColumnHeaderClick() {
+		this.getDomRef().querySelector(`#${this._id}-columnHeader`).focus();
```

Comment:
i wish we make this part of the item navigation but its fine for now

Reply:
I can try to cancat it with this.rows, it might work